### ISSUE TITLE
Docker log-opts and logrotate conflict

### ISSUE THREAD

**Issue by ialidzhikov** [gardener/gardener-extension-os-suse-jeos#6]
**What happened**:
The docker service on the nodes is configured with 
```
# cat /etc/docker/daemon.json
{
  "log-level": "warn",
  "log-driver": "json-file",
  "log-opts": {
    "max-size": "10m",
    "max-file": "5"
  }
}
```

which enables the log rotation from docker side. On the other hand we have also logrotate which will try to rotate the same container log files - https://github.com/gardener/gardener/blob/v1.0.4/charts/seed-operatingsystemconfig/original/templates/logrotate/docker.conf.
Now it is very likely to hit an issue caused by conflict between docker log rotation and logrotate.
Basically currently logrotate relies that the docker log rotation is not enabled.

**Environment**:

- Gardener version (if relevant): v1.0.4
- Extension version: v1.3.0
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- Others:
